### ISSUE TITLE
brats data result

### ISSUE THREAD

**Issue by 6018203135** [Amshaker/unetr_plus_plus#39]
Hello author, I'm really sorry, I want to ask a question about the ACDC dataset, the test set used by UNETR++ is 'patient089_frame01', 'patient089_frame10'
Does this count as one? However, there are only 10 imagesTs, the original ACDC dataset is 100, and the images Tr provided by the author have 80 (including 10 validation sets and 70 training sets)

**Comment by Amshaker** [comment]
It is duplicate of https://github.com/Amshaker/unetr_plus_plus/issues/38
